packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/bitmap.dart

```diff
@@ -29,6 +29,13 @@ enum MapBitmapScaling {
   none,
 }
 
+/// Convert a string from provided JSON to a MapBitmapScaling enum.
+MapBitmapScaling mapBitmapScalingFromString(String mode) => switch (mode) {
```

Comment:
This should be private so we're not increasing the API surface of the package.

Reply:
Would you prefer we mark it `@visibleForTesting`, or remove the test https://github.com/flutter/packages/pull/7699/files#diff-a8066076d538f1e1e21b27bac466302cb9317b643500ce81b991102a94611070R685?

Reply:
My preference would be visible for testing and keep the test.

Reply:
That would be my thought as well, @stuartmorgan do you agree with this decision?

Reply:
Yes, that's fine; from a semver standpoint we consider those equivalent.